README.md

```diff
@@ -6,10 +6,10 @@ The shared library manages all validation checks and constraints are defined in
 
 ## Installation
 
-TODO - Analyze value of pip support
+Using pip:
 
 ```
-pip install odevalidator
+pip install git+https://github.com/usdot-jpo-ode/ode-output-validator-library.git --upgrade
```

Comment:
Using GitHub path is good for packages that use the library but for someone who has cloned the library itself, they's want to build it from local files. SO I think this should be `pip install ./`